unittests/Feature/FeatureSourceRange.cpp

```diff
@@ -90,10 +90,10 @@ TEST(FeatureSourceRange, equality) {
 }
 
 TEST(FeatureSourceRange, clone) {
-  auto FSR = std::make_unique<FeatureSourceRange>(
+  auto FSR = std::make_unique<FeatureSourceRange>(FeatureSourceRange(
```

Comment:
Why do we need a direct ctor call here? This would then copy construct the FSR in the unique_ptr with the constructed FSR.